.github/workflows/publish-to-test-pypi.yml

```diff
@@ -0,0 +1,97 @@
+name: Publish Python 🐍 distribution 📦 to TestPyPI
+
+on:
+  workflow_dispatch:  # Keep manual trigger
+    inputs:
+      rc_version:
+        description: 'RC version number (e.g., 1, 2, 3)'
+        required: true
+        type: string
+  schedule:
+    - cron: "0 0 * * *"  # Run every day at midnight
+
+jobs:
+  trigger-client-build:
+    name: Trigger llama-stack-client build
```

Comment:
is the idea that we will not trigger llama-models workflow?

Reply:
I'm currently keeping llama-models as a separate package (since I think llama-models a version of llama-stack don't necessarily need to be aligned with llama-models version all the time).  

Or do you think we should add a trigger for llama-models to keep versions aligned?

Reply:
added llama-models trigger, now the workflow builds llama-stack which have dependencies on the same llama-models / llama-stack-client version packages. 

<img width="460" alt="image" src="https://github.com/user-attachments/assets/e16b10e8-4bab-4de7-a42f-f967a7acd3ee" />